src/convertToObject.js

```diff
@@ -12,7 +12,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString.split(';');
+  const arrTrimed = arr.map(prop => prop.trim());
+  const propsWithoutEmptyStrings = arrTrimed.filter(prop => {
+    if (prop !== '') {
+      return prop;
+    }
+  });
```

Comment:
you can use this trick to filter all empty values in the array
https://michaeluloth.com/filter-boolean

```suggestion
  const propsWithoutEmptyStrings = arrTrimed.filter(Boolean);
```